src/main/scala/redis/RedisPool.scala

```diff
@@ -35,13 +30,12 @@ abstract class RedisClientPoolLike(system: ActorRefFactory) extends RoundRobinPo
     }
   }
 
-  val redisConnectionRef: Ref[Seq[ActorRef]] = Ref(getConnectionsActive)
+  lazy val redisConnectionRef: Ref[Seq[ActorRef]] = Ref(getConnectionsActive)
```

Comment:
val redisConnectionRef: Ref[Seq[ActorRef]] = Ref(Seq.empty)
When we start we have no slave